### ISSUE TITLE
Sire website

### ISSUE THREAD

**Issue by lohedges** [michellab/Sire#275]
It would be good to dynamically update the website and documentation using an approach similar to that of BioSimSpace, i.e. sphinx and GitHub pages. This will require splitting @chryswoods personal website from the Sire documentation.

**Comment by kexul** [comment]
The website really need to update, I can't find the entrance of somd tutorial from the homepage. The only way to get there is google's search result by keyword 'somd tutorial'...

**Comment by jmichel80** [comment]
Agreed, this hasn't been high priority as we are moving to getting people to use somd via an interface provided by BioSimSpace. We are in the process of developing tutorials to describe the process. This is work in progress, you can follow developments here...https://github.com/michellab/BioSimSpaceTutorials

**Comment by kexul** [comment]
Many thanks for your effort, it would be a big step towards democratizing molecular simulation.
